components/ntp_background_images/browser/view_counter_service_unittest.cc

```diff
@@ -765,9 +760,6 @@ TEST_F(ViewCounterServiceTest,
        DoNotGetNewTabTakeoverWallpaperOnCuspOfGracePeriodForNonRewardsUser) {
   prefs_.SetBoolean(brave_rewards::prefs::kEnabled, false);
 
-  local_state_.SetInt64(metrics::prefs::kInstallDate,
-                        base::Time::Now().InSecondsFSinceUnixEpoch());
-
```

Comment:
Not needed because we now correctly set `metrics::prefs::kInstallDate` in SetUp.